src/images/local_images.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+from geopy import Point
+from geopy.distance import ELLIPSOIDS, distance
+from loguru import logger as log
+import numpy as np
+from PIL.ExifTags import GPS
+from PIL.Image import open
+from typing_extensions import override
+
+from src.images.image_source import ImageSource
+
+EXIF_GPS_TAG = 34853
+
+
+class LocalImages(ImageSource):
+    def __init__(self, images_path: Path) -> None:
+        """
+        All Args Constructor
+        :param images_path: Where the Images Should Be Located
+        """
+        super().__init__(images_path)
+        dir_images = (
+            set()
+            .union(images_path.glob("**/*.jpg"))
+            .union(images_path.glob("**/*.jpeg"))
+        )
+        if len(dir_images) == 0:
+            raise FileNotFoundError(f"No Images Found In Path: {images_path}")
+
+        self.images = dict()
+        for image_path in dir_images:
```

Comment:
Is this loop slow enough for a `tqdm` progress bar to be useful?

Reply:
On my computer the `__init__` function takes a second, so it probably doesn't need a `tqdm`.